12.miscellaneous/03.plurals/docs.en.md

```diff
@@ -48,100 +52,142 @@ Let’s draw a small parallel between English and Polish for better understandin
+<a name="3type"></a>
+#### 3. **Strings that use the Crowdin-developed pattern** 
 
-![](sign%26placeholders.png)
+This is the most user-friendly scheme of translating strings with plurals.
 
-Besides, almost all such strings have a yellow note above them which аttracts your attention to the proper use of plural forms.
+Crowdin suggests users to translate the certain amount of phrases with different plural forms. 
 
-Now, let’s imagine — you are faced with such phrase: **"Standard license for %count% computer|Standard license for %count% computers"**. 
+If your language provides only one plural form, then you will see only one phrase to translate. If you have three forms, Crowdin will provide you with the string divided into other two or three strings to translate.
 
-What should you know in order to translate it correctly?
+For example:
 
-The first thing you should understand is that the sentences which are divided by a vertical bar sign are called **'templates'** because they serve as templates for phrases with different numbers. The templates’ count usually starts with '0'.
+![crowdin](https://cdn.adguard.com/public/Adguard/kb/en/plurals/crowdin_scheme.png) 
+
+Please be attentive when translating these strings and approving them. If you have no idea what the "Other" field means, then just paste the same form as it is in the "Many" field. The "Many" and "Other" fields' contents can be the same.
+
+<a name="4type"></a>
+#### 4. **Strings with templates separated by the vertical bar sign**
+
+This is the most complicated type of AdGuard strings which is mostly used in `AdGuard Websites` project (namely in `"adguard.com"` folder).
+
+To identify phrases which translations require the use of plural forms you should pay your close attention to the presence of a vertical bar sign between the sentences and **%count%** placeholders in original phrases.
+
+![crowdin](https://cdn.adguard.com/public/Adguard/kb/en/plurals/plurals_site.png) 
+
+Now, let’s imagine — you are faced with such phrase: *"Standard license for %count% computer|Standard license for %count% computers"*. 
+
+What should you know to translate it correctly?
+
+Let's call sentences which are divided by a vertical bar sign the ***"templates"***, because actually they serve as templates for phrases with different numbers. 
 
 Coming back to the example, as the English language includes only two plural forms, there should be two templates respectively:
 
-**Template 0** – Standard license for *%count%* computer  
-**Template 1** – Standard license for *%count%* computers 
+`**Standard license for *%count%* computer|Standard license for *%count%* computers**`
 
-Another important thing you should pay your attention to is **%count%** placeholders that, as usual, take place before the words they define. Instead of **%count%** here will appear different numbers depending on which templates are being chosen.
+where **Template 1** is – Standard license for *%count%* computer,  
```

Comment:
Тут говорится про `Template 1` и `Template 2`, но при этом про них не упоминается раньше нигде

Reply:
Ещё `—` обычно заменяет собой глагол `to be`, а тут они вместе.

Reply:
а там двумя строчками выше есть вот такое предложение - Let's call sentences which are divided by a vertical bar sign the ***"templates"***, because actually they serve as templates for phrases with different numbers.